Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubis (2.7.0)
-    event_sourcery (0.13.0)
-    event_sourcery-postgres (0.3.0)
-      event_sourcery (>= 0.10.0)
+    erubi (1.7.1)
+    event_sourcery (0.22.0)
+    event_sourcery-postgres (0.8.0)
+      event_sourcery (>= 0.14.0)
       pg
-      sequel (~> 4.38)
-    highline (1.7.8)
-    method_source (0.8.2)
-    mustermann (1.0.0)
+      sequel (>= 4.38)
+    highline (2.0.0)
+    method_source (0.9.2)
+    mustermann (1.0.3)
     pg (0.20.0)
-    pry (0.10.4)
+    pry (0.12.2)
       coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
-    pry-byebug (3.4.2)
-      byebug (~> 9.0)
+      method_source (~> 0.9.0)
+    pry-byebug (3.6.0)
+      byebug (~> 10.0)
       pry (~> 0.10)
-    rack (2.0.3)
-    rack-protection (2.0.0)
+    rack (2.0.6)
+    rack-protection (2.0.4)
       rack
-    rack-test (0.6.3)
-      rack (>= 1.0)
-    rake (12.0.0)
-    rspec (3.6.0)
-      rspec-core (~> 3.6.0)
-      rspec-expectations (~> 3.6.0)
-      rspec-mocks (~> 3.6.0)
-    rspec-core (3.6.0)
-      rspec-support (~> 3.6.0)
-    rspec-expectations (3.6.0)
+    rack-test (1.1.0)
+      rack (>= 1.0, < 3)
+    rake (12.3.1)
+    rspec (3.8.0)
+      rspec-core (~> 3.8.0)
+      rspec-expectations (~> 3.8.0)
+      rspec-mocks (~> 3.8.0)
+    rspec-core (3.8.0)
+      rspec-support (~> 3.8.0)
+    rspec-expectations (3.8.2)
       diff-lcs (>= 1.2.0, < 2.0)
-      rspec-support (~> 3.6.0)
-    rspec-mocks (3.6.0)
+      rspec-support (~> 3.8.0)
+    rspec-mocks (3.8.0)
```

Comment:
### [rspec-mocks](https://github.com/rspec/rspec-mocks)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.6.0 → 3.8.0

[_~~change-log~~, [source-code](http://github.com/rspec/rspec-mocks)_]





<details>
<summary>Commits</summary>

A change of **61** commits. See the full changes on [the compare page](https://github.com/rspec/rspec-mocks/compare/v3.6.0...v3.8.0).

These are the first 10 commits:
- (cd7adc9) [Bump version to 3.7.0.pre](https://github.com/rspec/rspec-mocks/commit/cd7adc9fe1c418c2fe04d32f0961c615333af7aa)
- (38015e1) [Updates for frozen-string-literal compatibility. (#1165)](https://github.com/rspec/rspec-mocks/commit/38015e1aa82bd418581db5df565ef369d43b16ad)
- (a4b0b7f) [Update changelog.](https://github.com/rspec/rspec-mocks/commit/a4b0b7f9852fa7d6950864ecd6f46a8b2ffea9ce)
- (47f53d3) [Allow hash matchers to work with Hash descendants (#1167)](https://github.com/rspec/rspec-mocks/commit/47f53d38c8a2e3ef1b2b16b51936476004db4c0f)
- (4847ef0) [Changelog for #1167.](https://github.com/rspec/rspec-mocks/commit/4847ef067b60e3725a9e4819b9d0ee0f5c5262eb)
- (d8e2d47) [Update README.md (#1169)](https://github.com/rspec/rspec-mocks/commit/d8e2d4778eeb4117bea8a1bc9135a44720e65505)
- (0820af7) [Updated travis build scripts (from rspec-dev) (#1172)](https://github.com/rspec/rspec-mocks/commit/0820af734f1a8926b0fee4fa4dd3e00ff70b4990)
- (2f64278) [Updated travis build scripts (from rspec-dev)](https://github.com/rspec/rspec-mocks/commit/2f642786a93129e3408d4dc4843fb38409b90f88)
- (8b010b7) [Merge pull request #1174 from rspec/update-travis-build-scri…](https://github.com/rspec/rspec-mocks/commit/8b010b7a774570ceca86fb7bf6b0cc9da9a28eb0)
- (fa91024) [Updated travis build scripts (from rspec-dev)](https://github.com/rspec/rspec-mocks/commit/fa910249324822f2ebaac9ab9b7c10a8f6b7b701)


</details>